gooddata-sdk/gooddata_sdk/catalog/identifier.py

```diff
@@ -114,3 +117,13 @@ class CatalogNotificationChannelIdentifier(Base):
     @staticmethod
     def client_class() -> builtins.type[DeclarativeNotificationChannelIdentifier]:
         return DeclarativeNotificationChannelIdentifier
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogDeclarativeAnalyticalDashboardIdentifier(Base):
+    id: str
+    type: str = attr.field(validator=value_in_allowed)
```

Comment:
If there is only one type in `DeclarativeAnalyticalDashboardIdentifier` then it is not necessary to have type here. See identifers above.